src/Resources/Locales/de_DE.axaml

```diff
@@ -331,11 +331,11 @@
   <x:String x:Key="Text.InProgress.Merge" xml:space="preserve">Merge request wird durchgeführt. Drücke 'Abbrechen' um den originalen HEAD wiederherzustellen.</x:String>
   <x:String x:Key="Text.InProgress.Rebase" xml:space="preserve">Rebase wird durchgeführt. Drücke 'Abbrechen' um den originalen HEAD wiederherzustellen.</x:String>
   <x:String x:Key="Text.InProgress.Revert" xml:space="preserve">Revert wird durchgeführt. Drücke 'Abbrechen' um den originalen HEAD wiederherzustellen.</x:String>
-  <x:String x:Key="Text.InteractiveRebase" xml:space="preserve">Interactiver Rebase</x:String>
+  <x:String x:Key="Text.InteractiveRebase" xml:space="preserve">Interaktiver Rebase</x:String>
   <x:String x:Key="Text.InteractiveRebase.Target" xml:space="preserve">Ziel Branch:</x:String>
   <x:String x:Key="Text.InteractiveRebase.On" xml:space="preserve">Auf:</x:String>
-  <x:String x:Key="Text.InteractiveRebase.MoveUp" xml:space="preserve">Hinaufschieben</x:String>
-  <x:String x:Key="Text.InteractiveRebase.MoveDown" xml:space="preserve">Hinunterschieben</x:String>
+  <x:String x:Key="Text.InteractiveRebase.MoveUp" xml:space="preserve">Hoch schieben</x:String>
+  <x:String x:Key="Text.InteractiveRebase.MoveDown" xml:space="preserve">Runter schieben</x:String>
```

Comment:
Agreed.
According to [Duden](https://www.duden.de/suchen/dudenonline/hochschieben) we can write it in one word - "**hochschieben**".
Unfortunately, "runterschieben" cannot be found in Duden, but in dict.cc so ... we could just go with dict.cc, right? 😉

Reply:
Haha 😆 
But one word does indeed look cleaner. Additionally I would say that the correct spelling is just what most people agree on, since we both agree on "runterschieben" this is now correct 🙃.